css/main.css

```diff
@@ -261,65 +273,100 @@ section#credits h1 {
   color: white;
 }
 
-ul.credits-column {
-  list-style: none;
-  padding: 0;
-  margin: 0;
-  
-  box-sizing: border-box;
-  text-align: center;
-
+section.credits-column {
   min-height: auto;
   padding: 1.5em;
-  background: #BBD5DA;
+  background: #bbd5da;
   border-top-left-radius: 0.5em;
   border-bottom-right-radius: 0.5em;
-  margin: 1.5em 0.5em;
+  margin-left: 0.5em;
+  margin-right: 0.5em;
+  margin-top: 0;
+  margin-bottom: 1.5em;
   /* avoid breaking the message box between columns */
   break-inside: avoid;
 
   animation: 1s slide-up ease;
   animation-fill-mode: both;
   animation-iteration-count: 1;
 }
+section.credits-column ul {
+  list-style: none;
+  padding: 0;
+  margin: 0;
 
-ul.credits-column:nth-of-type(1) { animation-delay: .5s; margin-top: 0; }
-ul.credits-column:nth-of-type(2) { animation-delay: 1s }
-ul.credits-column:nth-of-type(3) { animation-delay: 1.5s }
+  box-sizing: border-box;
+  text-align: center;
+}
 
-ul.credits-column li {
+section.credits-column:nth-of-type(1) {
```

Comment:
Main change: changed all credit sections from ul to section > ul
mainly for ease of styling the twitter art section while following semantic html practices